common-practices-tools/contribution/prodev-contrib-sprint.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Contrib First Professional Development Sprint
+---
+
+# Contrib First Professional Development Sprint
+
+This serves as a guide for conducting a professional development sprint to build a contributed projects or features following [Contrib First](./contrib-first.md).
+
+## Kickoff Announcement Example
+
+This example can be used as a template to paste into Slack.
+
+**Pro-dev sprint:** Contrib First Drupal Module Build
+**Purpose:** To get more people from all practice areas experienced in building a contributed Drupal module as [Contrib First](https://guidebook.civicactions.com/en/latest/common-practices-tools/contribution/contrib-first/).
+**Time commitment:** 2-4 hours per week for 4 weeks (5-10% of your time)
+**Who:** We need a team
```

Comment:
@swirtSJW these need to be in a bullet list or a block to show up in not all inline